providers/shared/entrances/validate/entrance.ftl

```diff
@@ -0,0 +1,40 @@
+[#ftl]
+
+[#macro shared_entrance_validate ]
+
+    [#assign allDeploymentUnits = true]
+
+    [#-- override the deployment group to get all deployment groups --]
+    [@addCommandLineOption
+        option={
+            "Deployment" : {
+                "Unit" : {
+                    "Name" : ""
+                },
+                "Group" : {
+                    "Name" : "*"
+                },
+                "Framework" : {
+                    "Name" : DEFAULT_DEPLOYMENT_FRAMEWORK
+                }
+            },
+            "Flow" : {
+                "Names" : [ "views" ]
+            },
+            "View" : {
+                "Name" : BLUEPRINT_VIEW_TYPE
+            }
```

Comment:
I don't think that is quite right your view shared_view_default_validate_config should have a type as well I think

Reply:
I created a new view type for `validate` and moved existing macros out of the blueprint view.

31e3353